libs/application/templates/passport/src/lib/PassportTemplate.ts

```diff
@@ -92,7 +121,7 @@ const PassportTemplate: ApplicationTemplate<
     },
   },
   mapUserToRole(_id: string, _application: Application): ApplicationRole {
-    return 'applicant'
+    return Roles.APPLICANT
```

Comment:
This is going to assign every user the role Applicant even if that user does not own the application. You'll need to check the applicant.id to avoid this like its done in most other applications.